ormin/ormin_postgre.nim

```diff
@@ -25,15 +26,17 @@ proc c_strtod(buf: cstring, endptr: ptr cstring = nil): float64 {.
 proc c_strtol(buf: cstring, endptr: ptr cstring = nil, base: cint = 10): int {.
   importc: "strtol", header: "<stdlib.h>", noSideEffect.}
 
-var sid {.compileTime.}: int
+var sid = 0
+
+template genId*: untyped =
+  inc(sid)
+  sid
 
 proc prepareStmt*(db: DbConn; q: string): PStmt =
-  static:
-    inc sid
-    const name = "ormin" & $sid
```

Comment:
I think the ``const`` idea can be kept if you make it a ``.compileTime`` var and avoid the ``static`` block.

Reply:
If the const is kept the error is the following: 
`/ormin/ormin/ormin_postgre.nim(38, 15) Error: cannot 'importc' variable at compile time; pqprepare`

If let, or var is used instead the error is:
`/ormin/queries.nim(809, 43) Error: request to generate code for .compileTime proc: prepareStmt`